server/server.js

```diff
@@ -2,15 +2,63 @@ const express = require('express');
 
 const app = express();
 const port = 3001;
-// const db = require('../db/index.js');
+const db = require('../db/index.js');
 
 app.use(express.static('public'));
 
 app.get('/', (req, res) => {
   res.send('Hello World!');
 });
 
+// Get request to get all item details at particular item id.
+app.get('/api/items/:itemId', (req, res) => {
+  const id = req.params.itemId;
+
+  db.ItemDetails.find({ itemId: id }, (err, results) => {
+    if (err) {
```

Comment:
You should probably send a response to the client to let them know that the item was not found, otherwise the connection will timeout. This also applies to the other routes.